src/SFA.DAS.RoATPService.Database/StoredProcedures/RoATP_Audit_History.sql

```diff
@@ -0,0 +1,18 @@
+﻿CREATE PROCEDURE [dbo].[RoATP_Audit_History]
+AS
+	SELECT
+	o.UKPRN, 
+	o.LegalName AS [Legal Name],
+	ah.FieldChanged AS [Field Changed],
+	ah.UpdatedAt, 
+	ah.UpdatedBy, 
+	ah.PreviousValue AS [Old Value],
+	ah.PreviousStatusDate AS [Previous Status Date],
+	ah.NewValue AS [New Value],
+	ah.UpdatedAt AS [Change Date],
+	ah.UpdatedBy AS [Operator Name]
+	FROM [Audit] ah
+	INNER JOIN Organisations o 
+	ON o.Id = ah.OrganisationId
+RETURN 0
```

Comment:
This return is not needed, but don't mind if it stays.  Why the field name updates? Is this for some on-screen reportage?

Reply:
Yes, it's for inclusion in an Excel export